### PR TITLE
Task nxp 26845 merge web u plugin bis

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

**Comment by guirenard** [comment]
Tests passing https://qa.nuxeo.org/jenkins/job/Other/job/web-ui-pipeline-grenard/18/

**Comment by semisse** [comment]
👍

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
+metrics/
+node_modules/
+packages/nuxeo-web-ui-ftest/*
+!packages/nuxeo-web-ui-ftest/test
+plugin/
```
[the remark this answers is not in the document]
Comment (Gabez0r) [reply .eslintignore]: After this, 👍

**Comment by Gabez0r** [comment]
👍

**Pull request merged**